code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -30,31 +30,89 @@ var/global/list/parasites = list() //all currently existing/living guardians
 	maxHealth = INFINITY //The spirit itself is invincible
 	health = INFINITY
 	healable = FALSE //don't brusepack the guardian
-	damage_coeff = list(BRUTE = 0.5, BURN = 0.5, TOX = 0.5, CLONE = 0.5, STAMINA = 0, OXY = 0.5) //how much damage from each damage type we transfer to the owner
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 1, CLONE = 1, STAMINA = 0, OXY = 1) //1 by default but abilities chip down on the total 1.
```

Comment:
I advise it being 0.5 by default, so some increase it and some decrease it, so multiple stand abilities can't potentially be render a stand indestructible.